.github/workflows/build-test.yaml

```diff
@@ -31,3 +31,40 @@ jobs:
 
       - name: Build Tracee
         run: make build-docker
+  
+  test:
+    runs-on: ubuntu-20.04
+
+    env:
+      CGO_ENABLED: "1"
+
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15
+        id: go
+
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Lint
```

Comment:
Do we need the Lint here?
This is already done in "build" job

Reply:
Removed the Lint from "test" as suggested.